src/gui/mzroll/ligandwidget.cpp

```diff
@@ -394,6 +392,53 @@ void LigandWidget::showTable() {
     treeWidget->setSortingEnabled(true);
 }
 
+void LigandWidget::markAsDone(Compound* compound) {
+
+    QTreeWidgetItem* item = getItem(compound);
+
+    if (item) {
+        for (int col = 0; col < treeWidget->columnCount(); col++) {
+            item->setBackgroundColor(col, QColor(61, 204, 85, 100));
+        }
+    }
+}
+
+void LigandWidget::resetColor() {
+
+    QTreeWidgetItemIterator itr(treeWidget);
+    while (*itr) {
+        QTreeWidgetItem* item =(*itr);
+        if (item) {
+            for (int col = 0; col < treeWidget->columnCount(); col++) {
+                item->setBackgroundColor(col, QColor(255, 255, 255, 100));
+            }
+        }
+        ++itr;
+    }
+}
+
+QTreeWidgetItem* LigandWidget::getItem(Compound* compound) {
+
+    QTreeWidgetItem* matchedItem;
+    QTreeWidgetItemIterator itr(treeWidget);
+
+    while (*itr) {
+        QTreeWidgetItem* item =(*itr);
```

Comment:
Declare this outside the while loop. 
Having this inside would declare it for each time the loop runs